packages/python/plotly/_plotly_utils/tests/validators/test_angle_validator.py

```diff
@@ -3,6 +3,7 @@
 # from ..basevalidators import AngleValidator
 from _plotly_utils.basevalidators import AngleValidator
 import numpy as np
+from plotly.tests.b64 import b64
```

Comment:
I don't think you need to load this function.
Can't you simply pass numpy arrays in the tests below?

Reply:
Hm this fails without adding b64. Why do you think that is?

Reply:
I get `coerce_val = [1 2 3]`